edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Mqtt/MqttProtocolHead.cs

```diff
@@ -107,6 +109,8 @@ public async Task CloseAsync(CancellationToken token)
 
                 await (this.serverChannel?.CloseAsync() ?? TaskEx.Done);
                 await (this.eventLoopGroup?.ShutdownGracefullyAsync() ?? TaskEx.Done);
+                await (this.parentEventLoopGroup?.ShutdownGracefullyAsync() ?? TaskEx.Done);
```

Comment:
Can I bug you to change the logs to "Stopping MQTT protocol head" and "Stopped MQTT protocol head"?